sqrl-planner/src/main/java/com/datasqrl/actions/FlinkSqlGenerator.java

```diff
@@ -38,7 +38,10 @@ public Pair<List<String>, List<SqlNode>> run(StreamStagePlan physicalPlan) {
     flinkSql.addAll(result.getSinksSources());
     flinkSql.addAll(result.getQueries());
     if (result.getInserts().isEmpty()){
-      throw new RuntimeException("Flink stage empty. No queries or exports were found.");
+      throw new RuntimeException("No sinks detected."
+          + " Please configure an external sink or specify queries/subscriptions"
```

Comment:
Could also bring another engine

Reply:
you have have a sqrl job without "external sinks" (from the user perspective). Like a flink + postgres engine would bring a sink to postgres.